test/release/examples/primers/modules.chpl

```diff
@@ -514,14 +763,80 @@ module UserModule2 {
 }
 
 
-/* Since its use is ``public``, A scope with a ``use`` of ``UserModule2`` will 
+/* Since its use is ``public``, a scope with a ``use`` of ``UserModule2`` will
    also be able to see the symbols defined by ``ModuleThatIsUsed``.
 */
 module UsesTheUser2 {
   proc func2() {
     use UserModule2;
-    publiclyAvailableProc(); // available due to ``use`` of ModuleThatIsUsed
+    publiclyAvailableProc(); // available due to ``use`` of ``ModuleThatIsUsed``
+  }
+
+  // These lines are to ensure everything gets tested regularly
+  writeln("Start of UsesTheUser2's module-level code");
+  func2();
+  {
+    use UsesTheImporter;
   }
+  writeln("End of UsesTheUser2's module-level code");
 }
 
+/* Import statements can also be labeled as being either ``public`` or
+   ``private``.  By default ``import`` statements are also ``private``.
+   However, ``public import`` statements have a different meaning than ``public
```

Comment:
I am not sure if the difference is big enough to spell out explicitly in a primer. public uses and imports are can be considered almost identical w.r.t privacy most cases (except for tricky resolution order things), right? They both enable the symbols that they bring in to be brought in by use/import of the module that they are in.

Reply:
I think the main difference is in qualified naming for the symbols, but you're correct that I don't actually demonstrate that in this primer